MicroMasters data migration command and model

rhysyngsun opened the pull request:
#### Pre-Flight checklist

- [x] Migrations
  - [x] Migration is backwards-compatible with current production code
- [x] Testing
  - [x] Changes have been manually tested

#### What are the relevant tickets?
Fixes #721

#### What's this PR do?

- Adds data models for mapping course and program ids
- Adds a management command that runs SQL files in alphanumeric order by filename.

#### How should this be manually tested?

- You should be able to go to django-admin and add new records for `CourseId` and `ProgramId`
- You should be able to run the `./manage.py import_micromasters_data` command

rachellougee commented on micromasters_import/management/commands/import_micromasters_data.py:
```diff
@@ -0,0 +1,53 @@
+
+            for sql in sqls:
+                self.stdout.write(f"Executing SQL query: {sql.file_name}")
+                cursor.execute(sql.raw_sql)
```
If we could print out what's being executed in the sql file, that'd be helpful. It also would be nice to know how many rows are affected if it's insert using cursor.rowcount? I know that might get messed here. But we would want to get some insight of what's happening in these queries. 
Other than that, this import works with migration queries I had

rhysyngsun commented:
@rachellougee Addressed your feedback, I'm not sure exactly how the postgres driver determines the row count, but that property is what the documentation indicates should do the job.

rhysyngsun commented:
@rachellougee addressed your feedback, also noticed that `ProgramId` had an FK to `Course`, so I fixed that and redid the migration. You should run `./manage.py migrate micromasters_import zero` locally before pulling my latest changes otherwise you'll end up in a weird db state.

rachellougee reviewed (approved):
@rhysyngsun Re-tested. It looks good to me.
